extensions/amp-list/0.1/amp-list.js

```diff
@@ -927,27 +918,27 @@ export class AmpList extends AMP.BaseElement {
    * @private
    */
   changeToLayoutContainer_() {
-    const previousLayout = this.element.getAttribute('layout');
+    const previousLayout = this.element.getAttribute('i-amphtml-layout');
```

Comment:
When did we globally change the layout attribute to `i-amphtml-layout`? Our [documentation](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/#layout) still specifies the `layout` attribute.

Reply:
`i-amphtml-layout` is the one we set internally in `applyStaticLayout`. It's useful because you can omit the `layout` attribute sometimes, e.g.

```html
<!-- This is fixed layout despite not having [layout]. -->
<amp-img width=10 height=10 src=foo.jpg></amp-img>
```